externals/kyuubi-flink-sql-engine/src/main/java/org/apache/kyuubi/engine/flink/result/ResultSetUtil.java

```diff
@@ -18,41 +18,21 @@
 
 package org.apache.kyuubi.engine.flink.result;
 
-import java.util.ArrayList;
 import java.util.List;
 import org.apache.flink.table.api.DataTypes;
 import org.apache.flink.table.api.ResultKind;
 import org.apache.flink.table.catalog.Column;
-import org.apache.flink.table.types.DataType;
-import org.apache.flink.table.types.logical.VarCharType;
 import org.apache.flink.types.Row;
 
-/** Utility class for flink operation. */
-public class OperationUtil {
+/** Utility class for flink result sets. */
+public class ResultSetUtil {
```

Comment:
How about rewrite using Scala? Then we can avoid some `.asJava`

Reply:
Agreed.